esp32c3-hal/examples/ledc.rs

```diff
@@ -0,0 +1,72 @@
+#![no_std]
+#![no_main]
+
+use core::fmt::Write;
+
+use esp32c3_hal::{
+    clock::ClockControl,
+    gpio::IO,
+    pac::Peripherals,
+    prelude::*,
+    timer::TimerGroup,
+    RtcCntl,
+    UsbSerialJtag,
+};
+use esp_hal_common::ledc::{
+    channel::{self, ChannelIFace},
+    timer::{self, TimerIFace},
+    LSGlobalClkSource,
+    LowSpeed,
```

Comment:
I don't think it particularly matters, because only _one_ speed is available on each chip other than the esp32. I think @JurajSadel chose `LowSpeed` arbitrarily as the speed that all of them support.